frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -126,26 +104,59 @@ function ReaderBookmark:addToMainMenu(menu_items)
         }
     end
     menu_items.bookmarks_settings = {
-        text = _("Bookmarks"),
+        text = _("Annotation list"),
```

Comment:
"Item" is shorter than "annotation", it is used in the dialog headers.
We remember that we are in the Annotation list settings.

But - I do not insist and agree with any decision (the recent neverending wording discussions are a little bit exhausting).

Reply:
As discussed, I plan to change all "Bookmark"s to "Annotation"s (except "page bookmarks" of course).

Reply:
I'm a bit confused by the apparent plan to change all "bookmarks" to "annotations", including just highlighted sections. So then what is an underlined section with added note? A "noted annotation"? Or an "annotated annotation"? 

And if the Bookmarks list will be renamed to "Annotations", will page bookmarks no longer be listed there?

Reply:
Annotations are: page bookmarks, highlights, notes.
"Note" is a highlight with a note.

Reply:
> As discussed, I plan to change all "Bookmark"s to "Annotation"s (except "page bookmarks" of course).

So, may be keep this change here for the PR when you changed all the others?

I have no strong feeling about Bookmarks vs Annotation list - I'm just so used to Bookmarks, as I guess lots of other users, so it will be surprising. Better introduce this change later after next release.
(Just in case, we introduced ReaderAnnotations to clean some code mess - it does necessarily means the external names have to change and reflect the names used internally.)

Reply:
`Annotations` instead of `annotation list`